examples/meta/src/preprocessor/fish_lda.sg

```diff
@@ -0,0 +1,18 @@
+File data = csv_file("../../data/fm_train_real.dat")
+File lbls = csv_file("../../data/label_train_multiclass.dat")
+
+#![create_features]
+Features sg_features = features(data)
+Labels sg_labels = labels(lbls)
+#![create_features]
+
+#![create_pca_preprocessor]
+Transformer preproc = transformer("FisherLDA")
+preproc.put("final_dimensions", 1)
+preproc.put("method", enum EFLDAMethod.CANVAR_FLDA)
+preproc.fit(sg_features, sg_labels)
+#![create_pca_preprocessor]
+
+#![transform_features]
+Features featstrans = preproc.transform(sg_features)
```

Comment:
Just he one more look.
There is the thing that only numerical values/vectors etc are serialised for the testing, not the objects themselves. So in this example here, the transformed features (matrix) are not part of the test. But I think they should

Reply:
So if you extract the feature matrix, it will be serialised

Reply:
ok